waku/waku.md

```diff
@@ -510,6 +524,12 @@ Known static nodes MAY also be used.
 
 ## Changelog
 
+### Version 0.4
```

Comment:
In follow up commit let's add Release <date> with link as a signed commit and tag